packages/data-context/src/actions/WizardActions.ts

```diff
@@ -100,6 +100,14 @@ export class WizardActions {
 
     this.resetWizard()
 
+    await this.initializeFramework()
```

Comment:
Is this somehow related to this feature, or is it just a drive by bug fix?

Reply:
I pulled out the new `initializeFramework` action so we could trigger the existing "framework & bundler detection" logic while in e2e mode without touching other wizard stuff. I probably could have just used the existing `initialize` action since all it does is a reset then this logic, but I wanted to have a clean action in case there were updates to the wizard initialization flow in the future that we wouldn't want to fire while in e2e